base/compiler/ssair/domtree.jl

```diff
@@ -302,6 +313,7 @@ begin
             end
             idoms_dfs[v] = idom
         end
+        # The following produces for each BB in cfg, a map to its parent (as indexed in the cfg BB)
```

Comment:
Probably not incorrect, but really all this does is translate the idom relationship from dfs indexing to bb# indexing.

Reply:
Ah yes, so it is. I thought (but didn't write)
that this was also moving up 1 level, but `idoms` always was referring to up one level.

```suggestion
        # Reexpress the idom relationship in  BB indexing
```